pgrx/src/datum/array.rs

```diff
@@ -450,7 +440,7 @@ mod casper {
 
     /// Array elements are [`pg_sys::varlena`] types, which are pass-by-reference
     pub(super) struct PassByVarlena {
-        pub(super) align: usize,
+        pub(super) align: Align,
```

Comment:
Carrying a full `usize` is unnecessary here and in any case makes the result that much slightly-less typesafe.